src/serializer/MapFishPrintV3StamenSerializer.ts

```diff
@@ -1,9 +1,10 @@
 import OlLayer from 'ol/layer/Layer';
 import OlSource from 'ol/source/Source';
-import StadiaMaps from 'ol/source/StadiaMaps.js';
-import MapFishPrintV3OSMSerializer from './MapFishPrintV3OSMSerializer';
+import OlSourceStamen from 'ol/source/Stamen';
+import MapFishPrintV3XYZSerializer from './MapFishPrintV3XYZSerializer';
```

Comment:
did you test if the stamen layer still works when updating to openlayers 9.1? because there was a change recently with fixes for ol 9.1: https://github.com/terrestris/mapfish-print-manager/pull/468

Reply:
Apparently checks fail when running but IDE says everything is ok

Reply:
fixed in [83c1cb4](https://github.com/terrestris/mapfish-print-manager/pull/482/commits/83c1cb4080895eb0c35207f2e6116255a1bcc31e)